composer.json

```diff
@@ -12,8 +12,8 @@
     ],
     "require": {
         "php": ">=5.6.4",
-        "illuminate/queue": "5.3.*|5.4.*|5.5.*|5.6.*|5.7.*|5.8.*",
-        "illuminate/support": "5.3.*|5.4.*|5.5.*|5.6.*|5.7.*|5.8.*"
+        "illuminate/queue": "5.3.*|5.4.*|5.5.*|5.6.*|5.7.*|5.8.*|6.*",
```

Comment:
Laravel 6 and 7 use a different version selector than 5.x and below since it's now correctly semantic versioning. Could you change this to `^6.0`?

Also, it doesn't look like Laravel 7 has made changes to the Worker constructor, so would you be able to add `|^7.0` as well, to take care of both at once?